source/settings/plugin/quota-clone-plugin.rst

```diff
@@ -11,28 +11,33 @@ quota-clone plugin
 Settings
 ========
 
-.. dovecot_plugin:setting:: quota_clone_dict
+.. dovecot_plugin:setting_filter:: quota_clone
+   :filter: quota_clone
    :plugin: quota-clone
-   :values: @string
+   :setting: dict_driver
+   :values: @named_filter
+   :added: 2.4.0,3.0.0
 
-   The dictionary to update with quota clone information. This must be set for
-   the plugin to be active.
+   Named filter for initializing dictionary used to update with quota clone
+   information.
 
    See :ref:`dict` for dictionary configuration.
 
-   Example:
+   Options:
 
-   .. code-block:: none
+   ``unset``
```

Comment:
This isn't an "option". It's a global setting like everything else, so it should be specified as an independent global quota_clone_unset setting. It could be linked to here though.